src/components/ToDo.jsx

```diff
@@ -0,0 +1,69 @@
+import React from "react";
+import { useState, useEffect } from "react";
+import Button from "../elements/button/index.jsx";
+import Input from "./input/index.jsx";
+import TaskList from "./taskList/index.jsx";
+import '../App.css';
+import '@fortawesome/fontawesome-free/css/all.min.css';
+
+
+function ToDo() {
+    const [task, setTask] = useState('');
+    const [tasks, setTasks] = useState([]);
+
+    useEffect(() => {
+        const storedTasks = JSON.parse(localStorage.getItem('tasks')) || [];
+        setTasks(storedTasks);
+    }, []);
+
+    useEffect(() => {
+        localStorage.setItem('tasks', JSON.stringify(tasks));
+    }, [tasks]);
+
+    const handleChange = (event) => {
```

Comment:
This function name can be a little confusing.

How about naming it `handleInputChange`.